blockonomics.php

```diff
@@ -733,6 +777,16 @@ public function updateSettings()
             'BLOCKONOMICS_TIMEPERIOD',
             Tools::getValue('BLOCKONOMICS_TIMEPERIOD')
         );
+        $underpaymentSlack = Tools::getValue('BLOCKONOMICS_UNDERPAYMENT_SLACK');
+        if (0 <= $underpaymentSlack && $underpaymentSlack <= 20){
```

Comment:
Lets move this validation to the end of updateSettings to avoid other settings not being updated when an incorrect underpayment slack is set.